index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+</head>
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+<link rel="stylesheet" href="styles.css">
+<body>
+    <header class="header">
+    <nav class="navbar">
+        <a href="#" class="nabar-brand"> My logo</a>
+        <i class="fa fa-bars" class="menu-icon"></i>
+        
+    </nav>
+    </header>
+
+    <section class="Mobile-view">
+        <div class="mobile-container">
+            <div class="info">
+                <p class="intro">Hey There. I'm alex.</p>
+                <p class="skill">I'm a Software Developer</p>
+                <p class="bio">
+                I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+don't hesitate to contact me.
+                </p>
+                <div class="social-icons">
+                    <ul>
+                        <li><a href=""><img src="images/Vector.svg" alt="github-icon"></a></li>
+                        <li><a href=""><i class="fa fa-linkedin"></i></a></li>
+                        <li><a href=""><i class="fa fa-angellist"></i></a></li>
+                        <li><a href=""><i class="fa fa-twitter"></i></a></li>
+                        <li><a href=""><img src="images/Vector.png" alt="medium-icon"></a></li>
+                    </ul>
+                </div>
+            </div>
+            <div class="first-page-deco-img"></div>
+        </div>
+    
+
+       
```

Comment:
- Hey, let's remove this empty spaces to format our HTML file in a professional way.

Reply:
### README FILE


- [ ] The license link is broken, lets fix this by adding the appropriate missing license to make the readme more professional.
- [ ] let's remove this placeholder text as it's been added to guide you on making updates to the readme and is not expected to be in the readme file after you've made an update to the readme. see the image:
![Screenshot from 2023-03-20 19-50-52](https://user-images.githubusercontent.com/59397159/226438319-48bf1549-3db1-4789-81b8-528acfba4974.png)